resources/views/movies/create.blade.php

```diff
@@ -1,58 +1,93 @@
-                    </div>
+                                <x-input class="border-pink-500 border bg-purple-800 bg-opacity-25"
+                                type="date"
+                                name="release_date"
+                                id="release_date"
+                                :value="old('release_date')"/>
+                            </div>
+                            <!--Poster-->
+                            <div>
+                                <x-label for="poster" value="Poster"/>
 
-                    <!--Poster-->
-                    <div>
-                        <x-label for="poster" value="Poster"/>
+                                <x-input class="border-pink-500 border bg-purple-800 bg-opacity-25"
+                                type="text"
+                                name="poster"
+                                id="poster"
+                                :value="old('poster')"/>
+                            </div>
+                            <!--Trailer-->
+                            <div>
+                                <x-label for="trailer" value="Trailer"/>
 
-                        <x-input type="text" name="poster" id="poster" :value="old('poster')"/>
-                    </div>
+                                <x-input class="border-pink-500 border bg-purple-800 bg-opacity-25"
+                                type="text"
+                                name="trailer"
+                                id="trailer"
+                                :value="old('trailer')"/>
+                            </div>
+                        </div>
 
-                    <!--Trailer-->
-                    <div>
-                        <x-label for="trailer" value="Trailer"/>
+                        <!--cell 2-->
+                        <div class="justify-self-center grid grid-cols-1">
+                            <!--Blurb-->
+                            <div class="self-end">
+                                <x-label for="blurb" value="Blurb"/>
 
-                        <x-input type="text" name="trailer" id="trailer" :value="old('trailer')"/>
-                    </div>
+                                <textarea class="leading-normal border-pink-500 border bg-purple-800 bg-opacity-25"
+                                name="blurb"
+                                id="blurb"></textarea>
+                            </div>
+                        </div>
 
-                    <!--Blurb-->
-                    <div>
-                        <x-label for="blurb" value="Blurb"/>
+                        <!--cell 3-->
+                        <div class="pb-8 justify-self-center">
+                            <!--Celebrities-->
+                            <x-label for="celebrity" value="Celebrities"/>
 
-                        <textarea name="blurb" id="blurb"></textarea>
+                            <textarea class="leading-normal border-pink-500 border bg-purple-800 bg-opacity-25"
```

Comment:
This should probably be a `checkbox` instead of a `textarea`. The celebrities will need to be associated with a movie, the value of the checkbox would be a celebrity_id.

Reply:
Ah okay, I wasn't sure what the best way would have been to associating the celebrity with a movie given the size that the data potentially could have been